### ISSUE TITLE
Grid Consumption, FeedIn and Load do not work.

### ISSUE THREAD

**Issue by dekiel123** [macxq/foxess-ha#260]
Grid Consumption, FeedIn and Load do not work. But Energy Generated and "Power" works.
![image](https://github.com/macxq/foxess-ha/assets/110169964/fe73ca5e-545b-4937-afe9-fe300b4c3447)
![image](https://github.com/macxq/foxess-ha/assets/110169964/a43c244c-9b76-490f-bd99-001e71a5e342)

![image](https://github.com/macxq/foxess-ha/assets/110169964/51d28f2d-bc30-4b1a-ad09-837f44d26af4)
![image](https://github.com/macxq/foxess-ha/assets/110169964/3ffd5bac-ff91-4c1b-b621-96cc820d08c5)

![image](https://github.com/macxq/foxess-ha/assets/110169964/ce92635b-e5ea-487d-a627-008850765b8f)
![image](https://github.com/macxq/foxess-ha/assets/110169964/963ec58d-99e5-4550-b11e-9044871d4736)
![image](https://github.com/macxq/foxess-ha/assets/110169964/450e5ae0-6620-43ed-8b3b-917d8708a00d)
![image](https://github.com/macxq/foxess-ha/assets/110169964/4d2e614d-2e84-4412-8c77-31b5cec2a2a1)

![image](https://github.com/macxq/foxess-ha/assets/110169964/911b2057-6d32-4a8c-9aa8-c05603c42014)

**Comment by FozzieUK** [comment]
They all work for me
![IMG_1624](https://github.com/macxq/foxess-ha/assets/113460294/79a3eeea-e81c-4c64-9280-47a75b69756b)
![IMG_1623](https://github.com/macxq/foxess-ha/assets/113460294/3167716b-e812-4f2e-85fe-94e0992e42f5)
![IMG_1622](https://github.com/macxq/foxess-ha/assets/113460294/8826340b-143f-4d37-af1f-113f392a4999)

These are stored in the cloud as statistics not power, if you look on the website at your statistics graph do they show any values ?

![IMG_1626](https://github.com/macxq/foxess-ha/assets/113460294/eb39e94d-305d-43c1-931c-333ef082aedb)

**Comment by dekiel123** [comment]
![image](https://github.com/macxq/foxess-ha/assets/110169964/1ff109fe-dfe5-416a-b70b-de562e8e4b03)
Hmm, what does that mean?

**Comment by FozzieUK** [comment]
I think i’d be tempted to drop an email to fox service and ask them to look at why you are not recording any stats, it might be a firmware update required in your datalogger

**Comment by FozzieUK** [comment]
@dekiel123 i'd be grateful if you feed back what Fox say is causing it 👍

**Comment by dekiel123** [comment]
The first contact ended with the information that the statistics tab is a plan for the future and currently nothing works except TotalYield......

**Comment by FozzieUK** [comment]
> The first contact ended with the information that the statistics tab is a plan for the future and currently nothing works except TotalYield......

I've never come across that before from Foxess, what inverter is this and which country are you based in ?

**Comment by dekiel123** [comment]
T6. Poland.
I currently sent them your screenshot asking why do people have full stats... I'm waiting for a reply.

**Comment by FozzieUK** [comment]
Do you have to have the RS485 meter fitted which Fox regard as optional? - if you have the meter installed the T series would have everything it needs to record load power, feedin and grid consumption, if this isn't fitted maybe that is the reason why they don't record the statistics.

**Comment by dekiel123** [comment]
Unfortunately, I don't know, I recently purchased this house with photovoltaics, I only installed Chint DTSU666 myself.

Today's response from another employee:
"This person probably has a hybrid inverter with energy storage.
The bars are already visible there.

On ongrid and string inverters, this data is not collected in bars, only the current graph for now.

For these inverters, the meter connection function was mainly intended for the export limit function or zero export to the grid.
